src/lib/expansion/accordion.ts

```diff
@@ -25,7 +26,10 @@ export class CdkAccordion {
   /** A readonly id value to use for unique selection coordination. */
   readonly id = `cdk-accordion-${nextId++}`;
 
-  /** Whether the accordion should allow multiple expanded accordion items simulateously. */
+  /** Stream that emits when expandAll or collapseAll is triggered. */
+  readonly expandCollapseAllSubject: Subject<boolean> = new Subject<boolean>();
```

Comment:
Mark this with an underscore to denote that it is internal-only and add a JsDoc description (specifically that the `boolean` denotes expanded/collapsed)